easybuild/easyconfigs/p/pftools/pftoolsV3-24.03.16-foss-2016a.eb

```diff
@@ -0,0 +1,22 @@
+easyblock='ConfigureMake'
+
+name = 'pftoolsV3'
+version = '24.03.16'
```

Comment:
@RvDijk I'd go with `20160324`, i.e. the format we usually go for which sorts properly